### DISCUSSION TITLE
PanelChrome: Option to Make Panel Titles Hidden

### DISCUSSION THREAD

**Discussion by mertbakir** [grafana/grafana#38206] [Dashboards, Panels & Visualizations]
**What would you like to be added**:

I'm requesting two options for hiding panel titles. First one is in each panel's settings and the second one is in dashboard settings. An option to hide and show all panel titles.

**Why is this needed**:

Some dashboards are just looks better without the title. We can leave the title field empty but this makes using the panel results in another panel (using dashboard as a data source) is hard.

So, I don't want to display the panel title but use unique consistent names for panels and re-use the panel results in other panels in a structured way. This is the main reason for this request, you may come with another solution.

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!

**Comment by jwavoet** [comment]
I created an issue for this: https://github.com/grafana/grafana/issues/120423
